### ISSUE TITLE
Get accurate temperature setpoints

### ISSUE THREAD

**Issue by b-rowan** [wilfredallyn/pyasic_tools#8]
Not sure where the temperature Hot and Dangerous setpoints are being gathered from in the demo app, but pyasic can get those for you, just call `await miner.get_config()`, then you can use `config.temperature` to get temperature related configuration data.  It isn't particularly well documented as the format is super new, but the source code is here - https://github.com/UpstreamData/pyasic/blob/master/pyasic/config/temperature.py

Basically:

```python
async def get_temp_setpoints(ip: str) -> tuple[str | None]:
    miner = await pyasic.get_miner(ip)
    config = await miner.get_config()
    return config.target, config.hot, config.danger
```
